<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinFOSWindow</class>
 <widget class="QMainWindow" name="MinFOSWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(72, 189, 207);</string>
        </property>
        <property name="text">
         <string>MinFOS平台</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget {
    background-color: #FFFFFF;
    border: none;
    outline: 0;
}

QListWidget::item {
    border: none;
    background-color: #FFFFFF;
    color: #333333;
    padding: 15px 15px;
    font-size: 15px;
    qproperty-alignment: AlignCenter;
}

QListWidget::item:selected {
    background-color: #F5F5F5;
    color: #1976D2;
    font-weight: bold;
    font-size: 14px;
}

QListWidget::item:hover:!selected {
    background-color: #F8F8F8;
}</string>
        </property>
        <item>
         <property name="text">
          <string>通信管理</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>变量管理</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>用户管理</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>可视化界面</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>调度器</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="MainImageWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="ComListWidget">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QListWidget {
    background-color: #FFFFFF;
    border: none;
    outline: 0;
}

QListWidget::item {
    border: none;
    background-color: #FFFFFF;
    color: #333333;
    padding: 15px 15px;
    font-size: 15px;
    qproperty-alignment: AlignCenter;
}

QListWidget::item:selected {
    background-color: #F5F5F5;
    color: #1976D2;
    font-weight: bold;
    font-size: 14px;
}

QListWidget::item:hover:!selected {
    background-color: #F8F8F8;
}</string>
              </property>
              <item>
               <property name="text">
                <string>HTTP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>TCP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UDP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ModbusTCP</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="ComWidget">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
gridline-color: rgb(0, 0, 0);</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page_4">
               <widget class="QTableWidget" name="tableWidget">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>30</y>
                  <width>901</width>
                  <height>351</height>
                 </rect>
                </property>
                <column>
                 <property name="text">
                  <string>新建列</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>新建列</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>新建列</string>
                 </property>
                </column>
               </widget>
              </widget>
              <widget class="QWidget" name="page_6">
               <widget class="QTextEdit" name="TCPServer_textEdit">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>440</y>
                  <width>861</width>
                  <height>271</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:transparent;</string>
                </property>
               </widget>
               <widget class="QPushButton" name="StartTCPserver_Button">
                <property name="geometry">
                 <rect>
                  <x>280</x>
                  <y>290</y>
                  <width>71</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="text">
                 <string>开启服务</string>
                </property>
               </widget>
               <widget class="QPushButton" name="StopTCPserver_Button">
                <property name="geometry">
                 <rect>
                  <x>530</x>
                  <y>290</y>
                  <width>71</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="text">
                 <string>关闭服务</string>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="page_7"/>
              <widget class="QWidget" name="page_5"/>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <widget class="QToolButton" name="toolButton">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>210</y>
            <width>38</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
